dusk-abi/src/panic_include.rs

```diff
@@ -9,6 +9,17 @@
 // This file is meant to be included by the `include!` macro, and is therefore
 // not defined as a module
 
+// Use `wee_alloc` as the global allocator.
+#[global_allocator]
+static ALLOC: dusk_abi::WeeAlloc = dusk_abi::WeeAlloc::INIT;
+
+#[allow(improper_ctypes_definitions)]
+#[alloc_error_handler]
+#[no_mangle]
+pub extern "C" fn oom(_: ::core::alloc::Layout) -> ! {
+    ::core::intrinsics::abort();
```

Comment:
Test and document

Reply:
This is suggested directly in the `WeeAlloc` documentation. They have some sort of error handling via traps as in `Wasmi` for example. 

See: https://github.com/rustwasm/wee_alloc/blob/master/example/src/lib.rs#L18-L37

We can maybe just copy the comments from there.